### ISSUE TITLE
Correct CMS product-wide link color contrast accessibility issues

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9916]
## Background

In the CMS except for content types, there was an issue with link color contrast detected during the accessibility audit. The standard link style is blue underline with a hover style of a light blue background and no underline.  This combination of blue does not pass the WCAG 2.0 AA contrast requirements.

Image of defective area:
<img width="745" alt="Screen Shot 2022-07-27 at 12 26 11 PM" src="https://user-images.githubusercontent.com/90111970/181310877-7794a174-8130-4089-8191-8024076a7484.png">

## Possible Solution

This style could be updated to use the standard VA.gov hover styles, which are also what the links within the content types are using (#212121) link color with a background color of rgba (0,0,0,.05) and underlined. 

## Accessibility Standard

WCAG version 2.0 AA, [Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

## Acceptance Criteria

- [x] Interaction designer determines best style for hover links 
- [x] Design is reviewed with Accessibility Lead
- [x] Document changes to links in the CMS DS and changelog
- [x] technical review
- [x] tech implementation ticket created

**Comment by maxx1128** [comment]
If I could add my two cents here: we should also at least consider making all hyperlinks underlined. According to the WCAG, [hyperlinks that aren't underlined need at least a 3:1 contrast ratio and a non-color difference in styling for hover and focus interactions](https://webaim.org/techniques/hypertext/link_text#underlining) so users can tell them apart from plain text. My opinion is it's simpler and easier to have them underlined at all times. But I understand if discussing this is for a separate ticket.

**Comment by EWashb** [comment]
thanks @maxx1128! We agree that all should be underlined. Don't worry- any work on this will not lose the underline. This will only be changing the color contrast while keeping the underline.

**Comment by EWashb** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VAgov-CMS-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/9916?planning-poker) with ZenHub @laflannery @cjterdi

**Comment by cjterdi** [comment]
Per our accessibility review for links, we have come up with the follow criteria. This applies to [9917](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9917#issuecomment-1197604590) and [9915](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9917#issuecomment-1197604590)

All standard links will be changed to the dark blue (004795)
hover/focus state link color will be darker blue (003e73)
We think having a background color would be beneficial as this would keep it as a similar format as the front end (they have a background color on hover)
The hover/focus state should be the lighter blue (e4eff9). This is currently the color on some CMS links, not all. It is also different than the VA.gov front end hover color
Footer links will remain white
Footer links on hover/focus will keep their current background color (e4eff9) but the text will be darker blue (003e73)
All links should have underline on default and then no underline on hover/focus
